airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -603,13 +619,13 @@ def collect(self):
                     task.dag_id,
                     task.task_id,
                     task.owners,
-                    task.state or "none",
-                    task.cadence or "none",
-                    task.severerity or "none",
-                    task.alert_target or "none",
-                    task.instant_slack_alert or "none",
-                    task.alert_classification or "none",
-                    task.sla_time or "none",
+                    task.state or "missing",
+                    task.cadence or "missing",
+                    task.severity or "missing",
+                    task.alert_target or "missing",
+                    task.alert_external_classification or "missing",
+                    task.alert_report_classification or "missing",
+                    task.sla_time or "missing",
```

Comment:
Should we make a constant for "missing"?

Reply:
👍